app/front/scripts/services/api.js

```diff
@@ -0,0 +1,119 @@
+/**
+ * Created by Ihor Borysyuk on 26.01.16.
+ */
+
+;(function (angular) {
+
+  var app = angular.module('Application');
+  app.factory('ApiService', ['$q', '_', 'downloader', function ($q, _, downloader) {
+    var url_api_host = 'http://s145.okserver.org/api/3';
```

Comment:
let's have the hostname as an environment variable so it can also be used in dev environment